app/views/bookmarks/_form.html.haml

```diff
@@ -1,6 +1,9 @@
 - if @bookmark.errors.any?
   = render 'error_messages'
 
+#routes{hidden: 'true'}
```

Comment:
the routes should be in their own partial and get rendered in `layouts/application`